src/axom/klee/IOUtil.cpp

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "axom/klee/IOUtil.hpp"
+
+#include "axom/inlet.hpp"
+#include "axom/klee/KleeError.hpp"
+
+namespace axom
+{
+namespace klee
+{
+namespace internal
+{
+std::vector<double> toDoubleVector(inlet::Proxy const &field,
```

Comment:
See other comment on Proxy vs Container here - but in particular, the method `get<std::vector<double>>()` is only valid for a `Proxy` that references a `Container` and not a `Field` or `Function`.

Also, `field` might be a confusing name for a `Proxy` that references an `inlet::Container` and not an `inlet::Field`